tests/openvino/test_modeling.py

```diff
@@ -121,7 +121,7 @@
 
 SEED = 42
 
-F32_CONFIG = {"INFERENCE_PRECISION_HINT": "f32"}
+F32_CONFIG = {"INFERENCE_PRECISION_HINT": "f32",  "KV_CACHE_PRECISION": "f32"}
```

Comment:
```suggestion
F32_CONFIG = {"INFERENCE_PRECISION_HINT": "f32",  "KV_CACHE_PRECISION": "f16"}
```
You can rename it to DEFAULT_INFERENCE_CONFIG to make it more meaningful